scripts/android_package.sh

```diff
@@ -128,6 +128,13 @@ print Y "Configuring the android build"
 if [[ "$ADJUST_SDK_TOKEN" ]]; then
   print Y "Use adjust config"
   ADJUST="CONFIG+=adjust"
+
+  command -v find >/dev/null 2>&1 || die "find must be installed to work with the Adjust SDK"
+  command -v sed >/dev/null 2>&1 || die "sed must be installed to work with the Adjust SDK"
+  rm -rf "android/src/com/adjust"
+  cp -a "3rdparty/adjust-android-sdk/Adjust/sdk-core/src/main/java/com/." "android/src/com/"
+  find "android/src/com/adjust" -name *.java -exec sed -i "" "s/javax.net.ssl.HttpsURLConnection/java.net.HttpURLConnection/g" {} \;
+  find "android/src/com/adjust" -name *.java -exec sed -i "" "s/ttps/ttp/g" {} \;
```

Comment:
No, this is not good :/ We need to replace https more carefully. It seems that we just need to apply this patch:

```
diff --git a/Adjust/sdk-core/src/main/java/com/adjust/sdk/AdjustLinkResolution.java b/Adjust/sdk-core/src/main/java/com/adjust/sdk/AdjustLinkResolution.java
index 668fa20..2317e3c 100644
--- a/Adjust/sdk-core/src/main/java/com/adjust/sdk/AdjustLinkResolution.java
+++ b/Adjust/sdk-core/src/main/java/com/adjust/sdk/AdjustLinkResolution.java
@@ -105,11 +105,11 @@ public final class AdjustLinkResolution {
                                           final int recursionNumber,
                                           final AdjustLinkResolutionCallback adjustLinkResolutionCallback)
     {
-        final URL httpsUrl = convertToHttps(urlToRequest);
+        final URL httpUrl = convertToHttp(urlToRequest);
         URL resolvedURL = null;
         HttpURLConnection ucon = null;
         try {
-            ucon = (HttpURLConnection) httpsUrl.openConnection();
+            ucon = (HttpURLConnection) httpUrl.openConnection();
             ucon.setInstanceFollowRedirects(false);
 
             ucon.connect();
@@ -126,7 +126,7 @@ public final class AdjustLinkResolution {
             }
 
             resolveLink(resolvedURL,
-                    httpsUrl,
+                    httpUrl,
                     recursionNumber + 1,
                     adjustLinkResolutionCallback);
         }
@@ -154,7 +154,7 @@ public final class AdjustLinkResolution {
         return false;
     }
 
-    private static URL convertToHttps(final URL urlToConvert) {
+    private static URL convertToHttp(final URL urlToConvert) {
         if (urlToConvert == null) {
             return urlToConvert;
         }
@@ -171,7 +171,7 @@ public final class AdjustLinkResolution {
 
         URL convertedUrl = urlToConvert;
         try {
-            convertedUrl = new URL("https:" + stringUrlToConvert.substring(5));
+            convertedUrl = new URL("http:" + stringUrlToConvert.substring(5));
         } catch (final MalformedURLException ignored) { }
 
         return convertedUrl;
diff --git a/Adjust/sdk-core/src/main/java/com/adjust/sdk/Constants.java b/Adjust/sdk-core/src/main/java/com/adjust/sdk/Constants.java
index c8f1a0e..2f07c2e 100644
--- a/Adjust/sdk-core/src/main/java/com/adjust/sdk/Constants.java
+++ b/Adjust/sdk-core/src/main/java/com/adjust/sdk/Constants.java
@@ -27,7 +27,7 @@ public interface Constants {
     String GDPR_URL = "https://gdpr.adjust.com";
     String SUBSCRIPTION_URL = "https://subscription.adjust.com";
 
-    String SCHEME = "https";
+    String SCHEME = "http";
     String AUTHORITY = "app.adjust.com";
     String CLIENT_SDK = "android4.28.5";
     String LOGTAG = "Adjust";
``

Reply:
Did you try to build with that patch?